frontend/packages/kubevirt-plugin/src/components/vm-templates/os-icons.tsx

```diff
@@ -36,3 +36,5 @@ export const getTemplateOSIcon = (template: TemplateKind): string => {
 };
 
 export const PinnedIcon = () => <StarIcon className="kv-pin-icon" />;
+
+export const RemovedPinnedIcon = () => <StarIcon className="kv-empty-pin-icon" />;
```

Comment:
```suggestion
export const RemovedPinnedIcon = () => <StarIcon className="kv-remove-pin-icon" />;
```
?

Reply:
or "kv-gray-pin-icon" ?

Reply:
since the gray in the pattenfly palettet is light black, thought it would be a better idea to name it "kv-remove-pin-icon" as you suggested first